src/algokit/core/sandbox.py

```diff
@@ -284,4 +284,4 @@ def fetch_indexer_status_data(service_info: dict[str, Any]) -> dict[str, Any]:
         return {"Status": "Error"}
 
 
-DOCKER_COMPOSE_VERSION_COMMAND = ["docker", "compose", "version", "--format", "json"]
+DOCKER_COMPOSE_VERSION_COMMAND = ["docker", "--version"]
```

Comment:
This is the docker version rather than docker compose, if we used this we'd need to change the version we look for (we look for compose 2.5+ from memory because that was the point it was stable with bug patches for things users of algokit would likely come across.